BoxBrowseSDK/BoxBrowseSDK/BOXFolderViewController.m

```diff
@@ -400,44 +400,65 @@ - (void)fetchItemsWithCompletion:(void (^)(NSArray *items))completion
     folderRequest.SDKIdentifier = BOX_BROWSE_SDK_IDENTIFIER;
     folderRequest.SDKVersion = BOX_BROWSE_SDK_VERSION;
     folderRequest.requestAllFolderFields = YES;
-    [folderRequest performRequestWithCompletion:^(BOXFolder *folder, NSError *error) {
-        if (error) {
-            [self didFailToLoadItemsWithError:error];
-            self.navigationController.toolbarHidden = YES;
+
+    void (^completionBlock)(NSArray *, BOOL, NSError *) = ^void(NSArray *items, BOOL fromCache, NSError *error) {
+        dispatch_async(dispatch_get_main_queue(), ^{
+            if (items.count > 0) {
```

Comment:
Fair, but is there a situation where we would successfully retrieve items while also receiving an error?